FreeRTOS_TCP_IP.c

```diff
@@ -527,6 +527,8 @@
  *
  * @note Sequence of calling (normally) :
  * IP-Task:
+ *		prvIPTask()						// The IP-task
+ *		prvCheckNetworkTimers()			// The regular checks
```

Comment:
Does this code still use tabs?

Reply:
Sorry yes, I found some tabs within the comments. I will remove them.